### DISCUSSION TITLE
Crowdfunding campaign for pre-assembled haxophones in progress

### DISCUSSION THREAD

**Discussion by jcard0na** [cardonabits/haxo-hw#31] [Announcements]
If you would like to have a Haxophone, but would rather skip the building part, you can get one via our [crowdfunding campaign at Crowd Supply](https://www.crowdsupply.com/cardona-bits/haxophone). 

> [!NOTE]
> The campaign will run until October 19th 2023.

**Comment by rogerallen** [comment]
Hi there, your link above should be updated to add a "-" https://www.crowdsupply.com/cardona-bits/haxophone

**Reply by jcard0na** [reply]
Thank you @rogerallen!

**Comment by jcard0na** [comment]
Campaign finished with 187% funding.  Yay!
